confgenerator/config.go

```diff
@@ -737,7 +790,7 @@ func (uc *UnifiedConfig) MetricsReceivers() (map[string]MetricsReceiver, error)
 	if uc.Combined != nil {
 		for k, v := range uc.Combined.Receivers {
 			if _, ok := uc.Metrics.Receivers[k]; ok {
-				return nil, fmt.Errorf("metrics receiver %q has the same name as generic receiver %q", k, k)
+				return nil, fmt.Errorf("metrics receiver %q has the same name as combined receiver %q", k, k)
```

Comment:
Can we add an invalid unit test case for this `metrics receiver %q has the same name as combined receiver` error?

Reply:
Done